oci/oci.go

```diff
@@ -17,6 +17,21 @@ import (
 	spec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+// BlobInfo represents the mutable properties of a blob
```

Comment:
I would prefer to have this in the content package, but I didn't see a way to do that without completely deprecating the `oci` package. Some parts of the oci package are relying on the Commit interface and the `Commit` interface is relying on this type.